### DISCUSSION TITLE
Google Search: search can not be refined/ reloaded

### DISCUSSION THREAD

**Discussion by tdalon** [mmistakes/minimal-mistakes#2858] [Q&A]
I use the latest version of mm in github pages - see here: https://tdalon.github.io/ahk/

I have implemented the Google search engine in _config.yml as explained in the documentation here: https://mmistakes.github.io/minimal-mistakes/docs/configuration/#google-custom-search-engine

The search works but after I have run a search, it is not possible to refine the search means edit and run another search with other keywords.
Moreover the UI looks not so good:
![image](https://user-images.githubusercontent.com/6458444/109952662-604ad400-7cdf-11eb-9868-36755aa16fa1.png)

See the top right below the search button.
I am missing the search bar/ input box to run another search/ refine the search.

Right now it is tricky to rerun another search.
I need to reload the page to come back to this:
![image](https://user-images.githubusercontent.com/6458444/109952948-bb7cc680-7cdf-11eb-96e5-c8489995ac89.png)

**Comment by mmistakes** [comment]
You sure you set up Google Search correctly and selected the "Results only" layout as per the documentation?

![image](https://user-images.githubusercontent.com/1376749/109985771-1f4cc280-7cd3-11eb-83a1-e6421d7dc1cc.png)

I just tested with the demo site and it looks correct to me, and different from what you have. Either you configured it differently than the documentation or you customized the theme somewhere that's interferring with the styling.

![Screen Shot 2021-03-04 at 10 20 34 AM](https://user-images.githubusercontent.com/1376749/109985961-4c00da00-7cd3-11eb-8125-5ae3f6c3ac0b.png)

**Comment by tdalon** [comment]
Gosh. so sorry about that. I was sure to have follow the doc by the letter. but I hacve now checked and the layout was wrong/ set to overlay instead of results only.
Great support. very fast and great theme in general. Thanks Michael.
